tutorials/ab3.md

```diff
@@ -10,7 +10,7 @@
 
 * [**Tracing code**](ab3TracingCode.md) - A tutorial on how to use Intellij IDEA to trace execution paths in AB3 code.
 * [**Adding a command**](ab3AddRemark.md) - A tutorial on how to introduce a new user command to the AB3 code.
-* [**Removing fields**](ab3RemoveFields.md) - A tutorial on how to safely remove a field of the `Person` class.
+* [**Removing fields**](ab3RemovingFields.md) - A tutorial on how to safely remove a field of the `Person` class.
```

Comment:
I think this is fixed in the master branch already? Check if your master branch is behind.

Reply:
yeaps, have merged the latest changes